src/main/java/edu/cornell/kfs/fp/document/CuDisbursementVoucherDocument.java

```diff
@@ -639,7 +641,6 @@ public void toCopy() throws WorkflowException {
 
         super.toCopy();
 
-        KNSGlobalVariables.getMessageList().clear();
```

Comment:
Just out of curiosity,  was clearing the message list causing part of the problem?